app/_kong_plugins/rate-limiting/index.md

```diff
@@ -50,60 +56,25 @@ search_aliases:
   - rate-limiting
 ---
```

Comment:
I can't comment directly on it, but I think the first FAQ doesn't really need to exist, the page answers that question